admin/translations/fr.json

```
@@ -97,6 +97,14 @@
           },
           "success": "Le code campagne a bien été modifié."
         }
+      },
+      "update-organizations-in-batch": {
+        "title": "Modifier en masse des organisations",
```

```suggestion
        "title": "Modification des organisations en masse",
```

En effet il est préférable que les titres soient des noms (Modification) et que les boutons soient des verbes (Importer).

Enfin il est plus efficace à la lecture de mettre « en masse » à la fin car l'oeil cherche ce qui est différenciant et on recherche « Modification des organisations » quand on recherche cette fonctionnalité.

❤️ Idéalement un dernier commit pourrait être ajouté pour normaliser ainsi tous les titres des autres sections de la page d'administration !